conans/client/cmd/user.py

```diff
@@ -2,48 +2,36 @@
 from conans.client.store.localdb import LocalDB
 
 
-def users_list(client_cache, registry, remote_name=None):
-    # List all users from required remotes
-    if remote_name:
-        remotes = [registry.remote(remote_name)]
-    else:
-        remotes = registry.remotes
-
+def users_list(localdb_file, remotes):
     if not remotes:
         raise ConanException("No remotes defined")
 
-    localdb = LocalDB(client_cache.localdb)
-    result = []
+    localdb = LocalDB(localdb_file)
+    ret = {"remotes": []}
     for remote in remotes:
-        prev_user = localdb.get_username(remote.url)
-        username = prev_user or "None (anonymous)"
-        result.append((remote.name, username))
-    return result
+        user_info = {}
+        user, token = localdb.get_login(remote.url)
+        user_info["name"] = remote.name
+        user_info["user_name"] = user
+        user_info["authenticated"] = True if token else False
+        ret["remotes"].append(user_info)
```

Comment:
I dont understand why returning the dict with "remotes" only key, instead of just the list.

Reply:
I'd say this is to prepare the path for improvement in the json output when we have an object with error. e.g.

```
{
    "error": False,
    "remotes": [ ..... ]
}
```

Reply:
Even though. This local method should return something that makes sense locally, it is the caller assembling the JSON the one with the responsibility to add the ``remotes`` item.